### ISSUE TITLE
Subset in new API

### ISSUE THREAD

**Issue by lozunoff** [google/fonts#2507]
Hello!

How can I change the current subset in the new API (https://fonts.googleapis.com/css2)?

In the previous version, we could do it like this:
`https://fonts.googleapis.com/css?family=Roboto+Mono&subset=cyrillic`

What about the new version?

**Comment by yellow1912** [comment]
Yup, I have the same question. Also, is https://github.com/typekit/webfontloader still recommended?

**Comment by kotofenum** [comment]
Spent some time looking for it as it was removed from the https://fonts.google.com and neither mentioned in CSS API v2.

**Comment by yellow1912** [comment]
Can someone confirm this? Looks like a huge change and they don't even bother mentioning it anywhere?

**Comment by hyvyys** [comment]
I'm having trouble getting this to work. I only get basic Latin no matter what I do.
